modules/vf-graphql-holochain/queries/agent.ts

```diff
@@ -47,5 +60,23 @@ export default (dnaConfig: DNAIdMappings, conductorUri: string) => {
       return ((await readAgent({ address: args.id })).agent) as Person
       // TODO: type check if person or organization and provide error if organization
     }),
+    agents: async (root, args: PagingParams): Promise<AgentConnection> => {
+      let agents = (await readAll(args)) as AgentConnection
+      agents.edges = agents.edges.map((agentEdge) => {
+        agentEdge.node['__typename'] = 'Person'
+        return agentEdge
+      })
+      return agents
```

Comment:
I'm wondering if we want to backlog this issue where gql expects a concrete typename, which is why I inject 'Person' here. There is a comment above regarding checking the `agentType` field and injecting that instead of hardcoding 'Person'

Reply:
> checking the `agentType` field and injecting that instead of hardcoding 'Person'

Perfect. If the internal values used in the backend are different you might have to translate them but I figure you're already well aware of that.

Reply:
Yeah so I think you could actually set `__typeName` to be equal to `.agentType` since they both expect 'Person' or 'Organization'